addons/Magic_Boxes/src/presenter.js

```diff
@@ -167,7 +167,9 @@ function AddonMagic_Boxes_create() {
 
                 var selectableElement = $(document.createElement('div'));
                 selectableElement.addClass(presenter.CSS_CLASSES.ELEMENT);
-                selectableElement.text(presenter.configuration.gridElements[row][column].toUpperCase());
+                var rawChar = presenter.configuration.gridElements[row][column];
```

Comment:
uwaga funkcjonalna - jeżeli stosujemy ES6 (arrow function z linii poniżej) to warto również stosować słowa kluczowe let i const

Reply:
ok, będę pamiętać